lib/htmlGenerator2.js

```diff
@@ -232,11 +233,15 @@ function generate(text, tocBuilders, data) {
 	if (data.processFootnotes !== false) {
 		md.use(markdownItFootnotes);
 	}
-	
+
 	if (data.processTaskLists !== false) {
 		md.use(taskLists, {label: true, labelAfter: false, enabled: true});
 	}
 
+	if (data.processSpan !== false) {
```

Comment:
1. Rename data.processSpan to data.processSpans (plural)
2. There are accompanying required changes in marked-it.js, just follow the example there of `processTaskLists`

Reply:
Done!